app/editor/src/features/reports/av-overview/OverviewGrid.tsx

```diff
@@ -10,24 +12,52 @@ import {
   FormikTextArea,
   FormikTimeInput,
   IAVOverviewInstanceModel,
+  IOptionItem,
   Row,
   Show,
 } from 'tno-core';
 
 import * as styled from './styled';
 
 export interface IOverviewGridProps {
+  /** whether line item is editable or not  */
   editable: boolean;
+  /** index for av overview item */
   index: number;
+  /** helps to filter out content related to the show/program */
+  seriesId?: number;
 }
 
 /** OverviewGrid contains the table of items displayed for each overview section. */
-export const OverviewGrid: React.FC<IOverviewGridProps> = ({ editable = true, index }) => {
+export const OverviewGrid: React.FC<IOverviewGridProps> = ({
+  editable = true,
+  index,
+  seriesId,
+}) => {
   const { values, setFieldValue } = useFormikContext<IAVOverviewInstanceModel>();
+  const [, { findContent }] = useContent();
 
+  const [clips, setClips] = React.useState<IOptionItem[]>();
   const eveningOverviewItemTypeOptions = castEnumToOptions(AVOverviewItemTypeName);
   const items = values.sections[index].items;
 
+  /** fetch pieces of content that are related to the series to display as options for associated clips */
+  React.useEffect(() => {
+    findContent({
+      seriesId: seriesId,
+      contentTypes: [],
+    }).then((data) =>
+      setClips(
+        data.items.map((c) => ({
```

Comment:
You can use thew `new OptionItem(...)` class instead of manually casting.

Reply:
Right! Will change all that up, also realized as I was falling asleep last night that I also need to filter by the date as well